values.yaml

```diff
@@ -60,3 +66,7094 @@ mdai-operator:
+                            "type": "prometheus",
+                            "uid": "$datasource"
+                          },
+                          "editorMode": "code",
+                          "exemplar": true,
+                          "expr": "sum(otelcol_otelsvc_k8s_pod_deleted${suffix}{job=\"$job\"}) by (job $grouping)",
+                          "format": "time_series",
+                          "hide": false,
+                          "interval": "$minstep",
+                          "intervalFactor": 1,
+                          "legendFormat": "Deleted: {{transport}} {{service_instance_id}}",
+                          "range": true,
+                          "refId": "C"
+                        }
+                      ],
+                      "title": "Pods",
+                      "type": "timeseries"
+                    }
+                  ],
+                  "title": "Kubernetes",
+                  "type": "row"
+                },
+                {
+                  "collapsed": false,
+                  "gridPos": {
+                    "h": 1,
+                    "w": 24,
+                    "x": 0,
+                    "y": 81
+                  },
+                  "id": 66,
+                  "panels": [],
+                  "title": "Documentation",
+                  "type": "row"
+                },
+                {
+                  "datasource": {
+                    "type": "prometheus",
+                    "uid": "$datasource"
+                  },
+                  "editable": true,
+                  "error": false,
+                  "gridPos": {
+                    "h": 3,
+                    "w": 24,
+                    "x": 0,
+                    "y": 82
+                  },
+                  "id": 45,
+                  "links": [],
+                  "options": {
+                    "code": {
+                      "language": "plaintext",
+                      "showLineNumbers": false,
+                      "showMiniMap": false
+                    },
+                    "content": "<a href=\"http://www.monitoringartist.com\" target=\"_blank\" title=\"Dashboard maintained by Monitoring Artist - DevOps / Docker / Kubernetes / AWS ECS / Google GCP / Zabbix / Zenoss / Terraform / Monitoring\"><img src=\"https://monitoringartist.github.io/monitoring-artist-logo-grafana.png\" height=\"30px\" /></a> | \n<a target=\"_blank\" href=\"https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/troubleshooting.md#metrics\">OTEL collector troubleshooting (how to enable telemetry metrics)</a> | \n<a target=\"_blank\" href=\"https://opentelemetry.io/docs/collector/scaling/\">Scaling the Collector (metrics to watch)</a> | \n<a target=\"_blank\" href=\"https://opentelemetry.io/docs/collector/internal-telemetry/\">Collector internal telemetry</a> | \n<a target=\"_blank\" href=\"https://www.otelbin.io/\">OTelBin (OpenTelemetry colle
```

Comment:
A lot of these above will need to be enabled. Refer to the `values_grafana.yaml` file for what is needed for dashboards. When I imported the changes to the prometheus file, I switched them to false based on the existing prometheus yaml, but toggled them back on in the grafana file.

Reply:
I dunno why github grabbed the wrong line here... but I was referring to a lot of the things that are explicitly disabled below the dashboard jsons

Reply:
yeah, I had a quick yaml that I used with `-f` to test but there could be a lot of `--set` in the instructions and/or a short yaml to use with `-f`
```yaml
kube-prometheus-stack:
  # Enable features required for dashboards
  kubernetesServiceMonitors:
    enabled: true
  kubelet:
    enabled: true
  kubeStateMetrics:
    enabled: true
  kube-state-metrics:
    enabled: true
  nodeExporter:
    enabled: true
  prometheusOperator:
    kubeletService:
      enabled: true
  alertmanager:
    serviceMonitor:
      selfMonitor: true
```

Reply:
swapped these options to `true` by default and provided a `without_grafana.yaml` to utilize with `-f`